x-pack/plugins/apm/public/context/UrlParamsContext/helpers.ts

```diff
@@ -67,68 +55,3 @@ export function getPathAsArray(pathname: string = '') {
 export function removeUndefinedProps<T extends object>(obj: T): Partial<T> {
   return pickBy(obj, (value) => value !== undefined);
 }
-
-export function getPathParams(pathname: string = ''): PathParams {
```

Comment:
The only thing remaining here was `processorEvent` and that was moved to the `useProcessorEvent` hook that lives with the KueryBar.